satellitevu/apis/otm.py

```diff
@@ -233,3 +243,274 @@ def get_price(
             method="POST", url=url, json={k: v for k, v in payload.items() if v}
         )
         return response.json()
+
+
+class OtmV2(AbstractApi):
+    """
+    Client interface to the OTM API located at
+    https://api.satellitevu.com/otm/v2/docs.
+    """
+
+    api_path = "otm/v2"
+    scopes = ["tasking:w"]
+
+    def post_feasibility(
+        self,
+        *,
+        contract_id: UUID,
+        coordinates: Union[Tuple[float, float], Tuple[float, float, float]],
+        date_from: datetime,
+        date_to: datetime,
+        day_night_mode: str = "day-night",
+        **kwargs,
+    ):
+        """
+        Creates a tasking feasibility request.
+
+        Args:
+            contract_id: Associated ID of the Contract under which the feasibility
+            request will be performed.
+
+            coordinates: An array of coordinates.
+
+            date_from: datetime representing the start date of the feasibility request.
+
+            date_to: datetime representing the end date of the feasibility request.
+
+            day_night_mode: The mode of data capture, defaults to "day-night".
```

Comment:
Would be good to document what possible values are allowed. `["day", "night", "day-night"]`?

Again, applies below to other docstrings.